app/src/main/java/fr/free/nrw/commons/CommonsApplication.java

```diff
@@ -306,7 +309,7 @@ private void updateAllDatabases() {
         SQLiteDatabase db = dbOpenHelper.getWritableDatabase();
 
         CategoryDao.Table.onDelete(db);
-        ContributionDao.Table.onDelete(db);
```

Comment:
So all our existing users Contributions data - on upgrade - will remain on disk forever?

Reply:
Fair point, I will delete the existing contributions table

Reply:
Done